com/uber/kafkaSpraynozzle/stats/StatsDReporter.java

```diff
@@ -0,0 +1,61 @@
+package com.uber.kafkaSpraynozzle.stats;
+
+import com.timgroup.statsd.NonBlockingStatsDClient;
+import com.timgroup.statsd.StatsDClient;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Statsd StatsReporter implementation
+ *
+ * To initialize this class you need to provide it the path to a .properties file containing:
```

Comment:
And if you had to provide a configuration file to `curl` every time you ran it instead of command line arguments, how often would you use it? :)

How about treating the string as a JSON blob, so you could do something like:

```
-d '{"host": "localhost", "port": 8125, "prefix": "prefix.to.prepend"}'
```

Also even less likely to be a valid filename.